sycl/test/extensions/inline_asm.cpp

```diff
@@ -0,0 +1,79 @@
+// UNSUPPORTED: cuda
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
My understanding is that we are moving tests to llvm-test-suite, which require additional (to vanilla llvm) HW or SW dependencies. Following this logic this test should be moved to llvm-test-suite as well. Am I missing anything?

Tagging @andykaylor.

Reply:
I have removed execution on GPU device. So the test verifies that compilation succeeded